doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -313,15 +317,61 @@ The Platform is assumed to provide configurable alerting and notification capabi
 | sec.mon.021 | Monitoring | The Platform **must** store security events locally if the external logging system is unavailable and shall periodically attempt to send these to the external logging system until successful. | [RA-1 6.3.7.3 "Where to Log"](./chapter06.md#6373-where-to-log) |
 
+<p align="center"><b>Table 2-12:</b> Reference Model Requirements: Monitoring and Security Audit Requirements</p>
 
-#### 2.2.6.8. Compliance with Standards (source [RM7.9.8](../../../ref_model/chapters/chapter07.md#798-compliance-with-standards))
+#### 2.2.6.9. Open Source Software (source [RM7.9.8](../../../ref_model/chapters/chapter07.md#798-open-source-sotfware))
 
 | Ref # | sub-category | Description |  Traceability |
 |---------|---------------|----------------|------------|
-| sec.std.012 | Standards | The Public Cloud Operator **must**, and the Private Cloud Operator **may** be certified to be compliant with the International Standard on Awareness Engagements (ISAE) 3402 (in the US: SSAE 16); International Standard on Awareness Engagements (ISAE) 3402. US Equivalent: SSAE16. | |
+| sec.oss.001 | Open source code **must** be inspected by tools with various capabilities for static and dynamic code analysis. |  |
+| sec.oss.002 | The CVE(Common Vulnerabilities and Exposures) **must** be used to identify vulnerabilities and their severity rating for open source code part of Cloud Infrastructure and workloads software.  | https://cve.mitre.org/ |
+| sec.oss.003 | A dedicated internal isolated repository separated from the production environment **must** be used to store vetted open source content. |  |
+
+<p align="center"><b>Table 2-13:</b> Reference Model Requirements: Open Source Software Security Requirements</p>
+
+#### 2.2.6.9. IaaC security (source [RM7.9.9](../../../ref_model/chapters/chapter07.md#799-iaac---secure-design-and-architecture-stage-requirements))
+
+**Secure Code Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.code.001 | SAST -Static Application Security Testing **must** be applied during Secure Coding stage triggered by Pull, Clone or Comment trigger. | Security testing that analyses application source code for software vulnerabilities and gaps against best practices. Example: open source OWASP range of tools.|
+
+<p align="center"><b>Table 2-14:</b> Reference Model Requirements: IaaC Security Requirements, Secure Code Stage </p>
+
+**Continuous Build, Integration and Testing Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.bld.003 | Container and Image Scan **must** be applied during the Continuous Build, Integration and Testing stage triggered by Package trigger. | Example: A push of a container image to a container registry may trigger a vulnerability scan before the image becomes available in the registry.  |
```

Comment:
@karinesevilla Can we break these into two requirements: one for container and the other for image?

Reply:
@karinesevilla Agree. Will you open an RM Issue? You can "Resolve" after Issue is opened. Thanks

Reply:
@pgoyal01 Issue#2482 created